source/includes/_install_js_spa_intro.md

```diff
@@ -25,57 +25,167 @@
 * When you re-load the page, you should see Wootric survey show up.
 * You are done.
 
-## Angular Step 1: Create Factory
+## AngularJS Example (using Factory)
 
 ```javascript
-<!-- begin Wootric code -->
-angular.module('wootricAngularApp')
-.factory('wootricFactory', function () {
-window.wootricSettings = {
-    email: 'nps@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+// 1. Load Wootric JS SDK
+<script type="text/javascript" src="https://cdn.wootric.com/wootric-sdk.js"></script>
+
+// 2. Create Factory
+app.factory('wootricFactory', function() {
+  window.wootricSettings = {
+    email: 'customer@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
     created_at: 1234567890, // TODO: The current logged in user's sign-up date as a 10 digit Unix timestamp.
-    account_token: 'NPS-xxxxxxx' // This is your unique account token.
+    account_token: 'NPS-xxxxxxxx' // This is your unique account token.
   };
   return {
     run: function() {
-      window.wootric_survey_immediately = true; // TODO: Comment out for production.      
+      window.wootric_survey_immediately = true; // Shows survey immediately for testing purposes.  TODO: Comment out for production.
+      window.wootric_no_surveyed_cookie = true; // Bypass cookie based throttle for testing purposes.  TODO: Comment out for production.     

```

Comment:
Why are we including this in the example? Do customers need it regularly?

Reply:
It's more like that we have those to test out quickly and as `TODO` says to comment those out in production.